src/views/docs/en/reference/cli/sandbox.md

```diff
@@ -61,6 +61,8 @@ Several Architect [local preferences](../configuration/local-preferences) can be
 
 The following can be set as a part of the [`@sandbox`](../configuration/local-preferences#%40sandbox) pragma.
 
+- `livereload` - Enable automatic reload for `@http` `get` and `any` functions. When a filesystem change is detected in the handler, `@shared`, or `@views` code, the function will be reloaded in open browser sessions.
```

Comment:
maybe: Enable live automatic reload for `@http` `get` and `any` functions that deliver HTML. When a filesystem change is detected in the handler or in [shared or views code](/docs/en/guides/developer-experience/sharing-code), open browser sessions will automatically refresh.